jest.integration.config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  collectCoverageFrom: [
+    '<rootDir>/shared/**/*.(js|ts|tsx)',
+    '<rootDir>/ui/**/*.(js|ts|tsx)',
+  ],
+  coverageDirectory: './coverage/integration',
```

Comment:
Are we concerned about coverage with integration tests?

My understanding was we're just using them to supplement E2E coverage, or are we actively trying to cover all pages?

Reply:
Yes I don't think that test coverage will add a ton of value here. The real value would be on use case coverage but there's no easy way to measure that. The though was, well better to had the coverage config for the code coverage that devs are used, than nothing. (I this coverage report is not being merged with any of the unit tests coverage reports neither being reported to any tool - like codecov or sonarcloud... for now it is just for local generation and support the developer adding integration tests)

Reply:
I would expect coverage to be useful here. We're currently striving to increase coverage as much as possible, with the goal of getting to 100%. But certain components (particularly page-level components) would be pointless to test in an isolationist way by mocking everything. The only useful way of testing it would be with an integration test. Considering integration test coverage would be one way of reducing pressure to write poor unit tests.

But we can figure this out in a later PR.